gotechnica/platform-2020
# update home page description

**Changes Made:**
- Changed page title
- Changed home page description 

**Addressed tickets:**
https://trello.com/c/umFe2RqW/217-navigation-update-home-page-description
https://trello.com/c/9uEL2I0V/216-navigation-update-frontend-page-title

**Steps to test:**
- Make sure page title says "Technica Virtual Platform"
- Make sure description under "Welcome to Technica's Virtual Platform!" says: "This is the Technica platform's home page (currently under development). Navigation to other pages is available through the links at the top of the page."
